### PR TITLE
Feat/roc 9571 discosure electronic and non-electronic screen

### PR DESCRIPTION
**Pull request by bhssamray**
### JIRA link (if applicable) ### 
https://tools.hmcts.net/jira/browse/ROC-9571

### Change description ###
This PR consists of screens for Electronic and non Electronic documents information capture in the Spec defendant response journey. Also contained is a fix for new spec camunda event added recently.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: ccd-definition/CaseEventToFields/DefendantResponseLRspec.json [hunk 1]
```diff
@@ -258,6 +258,7 @@
     "PageDisplayOrder": 6,
     "PageFieldDisplayOrder": 4,
     "DisplayContext": "READONLY",
+    "DisplayContext": "READONLY",
```
Comment (bhssamray) [inline ccd-definition/CaseEventToFields/DefendantResponseLRspec.json:261]: Check the duplicate entry of DisplayContext

Comment (bhssamray) [reply ccd-definition/CaseEventToFields/DefendantResponseLRspec.json]: Removed the duplicate attribute in object.

**Comment by HarryH96** [comment]
@bhssamray can you add relevant information to the PR for the changes that you are making + link the Jira ticket please :)

**Review by akriti-hmcts — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Comment by bhssamray** [comment]
> @bhssamray can you add relevant information to the PR for the changes that you are making + link the Jira ticket please :)

Sorry, added the details now. Thanks @HarryH96

**Pull request merged**